sample_shavar_list_creation.ini

```diff
@@ -127,6 +127,13 @@ output=mozplugin2-block-digest256
 [entity-whitelist]
 entity_url=https://s3.amazonaws.com/lists.disconnect.me/exampleservices.json
```

Comment:
This should be https://raw.githubusercontent.com/mozilla-services/shavar-prod-lists/master/disconnect-entitylist.json